app/controllers/admin/resources_controller.rb

```diff
@@ -23,12 +23,25 @@ def create
     end
 
     def update
-      if @resource.update_attributes(resource_params)
-        redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
-                    notice: 'Resource successfully updated.'
+      if params[:inc].present?
```

Comment:
What does inc mean? Please rename it to something that I can understand when reading the code.